src/Task/CopyDevPaths.php

```diff
@@ -124,12 +140,81 @@ private function copyCustomPath(string $pathConfigKey, Io $io): int
             return 0;
         }
 
+        return $this->copyFolder($sourcePaths, $target, $io);
+    }
+
+    /**
+     * @param Io $io
+     *
+     * @return int
+     */
+    private function copyMuPluginsPackages(Io $io): int
+    {
+        $rootPackage = $this->composer->getPackage();
+        $repositoryManager = $this->composer->getRepositoryManager();
+
         $errors = 0;
+        foreach ($rootPackage->getRequires() as $link) {
+            /** @var ConstraintInterface $constraint */
+            $constraint = $link->getConstraint();
+            if (!$constraint instanceof Constraint) {
```

Comment:
I am not sure if I need this check here or if I have to pass this variable to `findPackage()`.